utils/ziputil/zip.go

```diff
@@ -0,0 +1,150 @@
+// Copyright 2018 the Service Broker Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ziputil
+
+import (
+	"archive/zip"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/GoogleCloudPlatform/gcp-service-broker/utils/stream"
+)
+
+// List writes a ls -la style listing of the zipfile to the given writer.
+func List(z *zip.Reader, w io.Writer) {
+	sw := tabwriter.NewWriter(w, 0, 0, 2, ' ', tabwriter.StripEscape)
```

Comment:
It's generally good practice when using literal arguments to functions like this to annotate them with a comment with the parameter name they correspond to. Makes for easier code reading.

Reply:
I noticed this feedback was resolved but I don't see any related changes. Any thoughts?